framework/auth/views.py

```diff
@@ -134,8 +145,20 @@ def reset_password_post(uid=None, token=None):
     }
 
 
+def reset_password_institution_post(uid=None, token=None):
+    return reset_password_post(uid=uid, token=token)
+
+
+def forgot_password_get():
+    return _forgot_password_get()
+
+
+def forgot_password_institution_get():
+    return _forgot_password_get()
```

Comment:
Unlike the `reset_password_institution_post` and `reset_password_institution_get` that can be merged into `reset_password_post` and `reset_password_get` , this `forgot_password_institution_get()` and `GET /forgotpassword-institution/` route probably needs to be disabled.

This removes confusion in the future so we only have one "entry point", which is implemented in / has been moved to CAS. A better option can be redirecting to the CAS unsupported institution login page.

Here is the route definition to be removed.

```python
Rule(
    '/forgotpassword-institution/',
    'get',
    auth_views.forgot_password_institution_get,
    OsfWebRenderer('public/forgot_password_institution.mako', trust=False)
),
```

Here is what I see if I go to `http://localhost:5000/forgotpassword-institution/`

Reply:
Ooh, good catch!  Thanks for that, I will fix it up as suggested.